spec/opal/stdlib/json/parse_spec.rb

```diff
@@ -30,4 +30,8 @@
     JSON.parse('{"a": "b"}').should == {"a" => "b"}
     JSON.parse('{"a": null, "b": 10, "c": [true, false]}').should == {"a" => nil, "b" => 10, "c" => [true, false]}
   end
+
+  it "raises ParserError for invalid JSON" do
+    lambda { JSON.parse("invalid_json") }.should raise_error(JSON::ParserError, "Unexpected token i in JSON at position 0")
```

Comment:
Looks like it fails on Travis because of the platform-specific error message. With node `0.10.25` (I know, it's very old) and in chrome I'm getting:
``` js
try {
  JSON.parse('invalid_json')
} catch(e) {
  console.log(e.message)
}
// => Unexpected token i
```

Probably it's easier to leave only `should raise_error(JSON::ParserError)`